pkg/podexec/pod_command_executor_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 the Velero contributors.
+Copyright 2017, 2020 the Velero contributors.
```

Comment:
The modified printer files probably also need this copyright update, too.

Reply:
will do

Reply:
pushed new commit